README.md

```diff
@@ -290,7 +290,7 @@ const drive = google.drive({
 });
 ```
 
-See the [Options section][options] for more information.
+See the [Options section][request-options] for more information.
```

Comment:
I think this should be `[Options section](#request-options)`?

Reply:
I thought so too, but since the original worked like that I assumed it was a valid style. I'll make that update.